.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,37 @@
+name: Feature Request
+description: Suggest a new feature
+body:
+  - type: markdown
+    attributes:
+      value: >
+        **Attention:** if this is a _question_ about how to build / test / query / deploy using
+        Bazel intellij, or a _discussion starter_, send it to bazel-discuss@googlegroups.com or discuss on
```

Comment:
Also here we should direct the users to this group: [intellij-bazel-plugin@googlegroups.com](mailto:intellij-bazel-plugin@googlegroups.com)